apps/server/src/modules/account/controller/account.controller.ts

```diff
@@ -1,9 +1,12 @@
 import { Body, Controller, Delete, Get, Param, Patch, Query } from '@nestjs/common';
 import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { EntityNotFoundError, ForbiddenOperationError, ValidationError } from '@shared/common';
-import { ICurrentUser } from '@src/modules/authentication';
-import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator/auth.decorator';
+import { ICurrentUser } from '@modules/authentication';
+import { Authenticate, CurrentUser } from '@modules/authentication/decorator/auth.decorator';
```

Comment:
you removed the src/, but you didnt make sure to not import from deep within the module.
both Authenticate and CurrentUser is available in the index file of the authentication module, so this can be shortened to 
`import { ICurrentUser, Authenticate, CurrentUser } from '@src/modules/authentication';`